### PR TITLE
Add polyline component

### LINKED ISSUE #16: [Component] LPolyline
To migrate a component from Vue2Leaflet to vue-leaflet (and so from vue2 to vue3) follow this steps:

- create a function in `functions` folder called `componentName.js` (without the `L`)
- check what mixins are used in the vue2 version and see if all the relative composition API function are present, if not they need to be created
  - Each composition API function needs to export two objects: props and setup, see `marker.js` for an example.
- create a component in `components` folder called `LComponentName.vue` 
- follow the guide of `LMarker.vue` to understand how to deal with reactivity, dynamic import and SSR
- add and test the component inside `playground.vue`

### PR DESCRIPTION
**Pull request by DonNicoJs**
Fixes #16

### CODE REVIEW

**Comment by DonNicoJs** [comment]
@mikeu Would you take a look here? :)

File: src/components/LPolyline.vue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      propsBinder(methods, leafletRef.value, props, setOptions);
+
+      addLayer({
+        ...props,
+        ...methods,
+        leafletObject: leafletRef.value,
+      });
+      ready.value = true;
+    });
+    return { ready };
```
Comment (mikeu) [inline src/components/LPolyline.vue:39]: What is the reason to return the `ready` value, or in fact define it at all? As far as I can see it's never used in the polyline logic, and it's definitely not used in the render template either.

**Review by mikeu — APPROVED** [review]

File: src/functions/polyline.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { props as pathProps, setup as pathSetup } from "../functions/path";
```
Comment (mikeu) [inline src/functions/polyline.js:1]:
```suggestion
import { props as pathProps, setup as pathSetup } from "./path";
```

File: src/components/LPolyline.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    onMounted(async () => {
+      const { polyline, DomEvent, setOptions } = await import(
+        "leaflet/dist/leaflet-src.esm"
+      );
+
+      leafletRef.value = polyline(props.latLngs, options);
+
+      const listeners = remapEvents(context.attrs);
+      DomEvent.on(leafletRef.value, listeners);
```
Comment (mikeu) [inline src/components/LPolyline.vue:28]: Should we also be calling `DomEvent.off` on this and other components, when they unmount?

File: src/components/LPolyline.vue (hunk 1 above)
Comment (DonNicoJs) [reply src/components/LPolyline.vue:39]: @mikeu you are right! The component in the v2 was using it but I thought it did not made much sense, upon further thought I realise that a `polyline` is infact a layer... so you may want to nest something in it, addressing this now!

Comment (DonNicoJs) [reply src/components/LPolyline.vue:39]: Fixed :)

File: src/components/LPolyline.vue (hunk 3 above)
Comment (DonNicoJs) [reply src/components/LPolyline.vue:28]: @mikeu 🤔  probably yes, I'll open an issue to tackle this on all the components

**Comment by DonNicoJs** [comment]
@mikeu Would you mind taking another look? :)

**Review by mikeu — APPROVED** [review]
LGTM!

**Pull request merged**